<launch>

	<rosparam command="load" file="$(find wall_follow)/launch/columnLoopUp_params.yaml"/>
	<node pkg="wall_follow" type="wall_follow_columnLoop" name="columnLoop" ns="up">
		<remap from="/up/mavros/local_position/pose" to="/mavros/local_position/pose"/>
		<remap from="/up/mavros/state" to="/mavros/state"/>
		<remap from="/mavros/setpoint_raw/local" to="/up/columnLoopPub"/>
	</node>

</launch>